packages/plugins/rubygem-foreman_openscap/rubygem-foreman_openscap.spec

```
@@ -92,6 +93,9 @@ cp -a .%{gem_dir}/* \
 %{foreman_plugin_log}
 
 %changelog
+* Wed Jul 19 2023 Leos Stejskal <lstejska@redhat.com> 7.0.0-2
+- Revert dependency back for scap-security-guide
```

Revert back sounds redundant. Would this be clearer?
```suggestion
- Add dependency back for scap-security-guide
```